src/NECompletion/NECContext.class.st

```diff
@@ -267,13 +268,23 @@ NECContext >> findSourceRangeFor: aNumber [
 										ifFalse: [^ nil]]]]]
 ]
 
+{ #category : #accessing }
+NECContext >> globals [
+	^ globals
+]
+
+{ #category : #accessing }
+NECContext >> globals: anObject [
+	globals := anObject
+]
+
 { #category : #'private-type-guessing' }
 NECContext >> guessArgument: aSHRange [ 
 	| name |
 	name := self sourceOf: aSHRange.
 	(name = 'html') ifTrue: [
```

Comment:
It's not a problem with this PR but... Why do we have Seaside related code directly in Pharo?! Maybe a hook is missing in the code completion for other frameworks to plug their behavior?

Reply:
🗡 👯‍♂️ 🍡